src/atomate2/aims/schemas/calculation.py

```diff
@@ -94,7 +94,7 @@ class CalculationOutput(BaseModel):
         None, description="The final structure from the calculation"
     )
 
-    efermi: float = Field(
```

Comment:
Some of these changes might be in the mp repository, possible conflict when merging in